docs(flutter_todos): include barrel files in tutorial for completeness (#4350)
felangel/bloc: docs/src/content/docs/tutorials/flutter-todos.mdx

The model of the `Todo` is defined in `todos_api/models/todo.dart` and is exported by `package:todos_api/todos_api.dart`.

#### Update Exports

> Our ToDo model and the ToDosApi is made available externally (and internally) through barrel files. Notice how we don't import the ToDo model directly, but we import it in `lib/src/todos_api.dart` with a reference to the package barrel file: `import 'package:todos_api/todos_api.dart';`. Now, update the barrel files to resolve any remaining import errors.

Reviewer: ```suggestion
Our `Todo` model and the `TodosApi` are exported via barrel files. Notice how we don't import the model directly, but we import it in `lib/src/todos_api.dart` with a reference to the package barrel file: `import 'package:todos_api/todos_api.dart';`. Update the barrel files to resolve any remaining import errors:
```
